Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -30,17 +43,12 @@ final class PledgeCTAContainerView: UIView {
     _ = ([self.youreABackerLabel, self.amountOrRewardLabel], self.amountAndRewardTitleStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    _ = ([self.amountAndRewardTitleStackView, self.pledgeCTAbutton], self.rootStackView)
+    _ = ([self.amountAndRewardTitleStackView, self.spacer, self.pledgeButton], self.rootStackView)
       |> ksr_addArrangedSubviewsToStackView()
 
-    NSLayoutConstraint.activate(
-      [
-        self.pledgeCTAbutton.heightAnchor.constraint(
-          greaterThanOrEqualToConstant: Styles.minTouchSize.height
-        ),
-        self.pledgeCTAbutton.trailingAnchor.constraint(equalTo: self.rootStackView.trailingAnchor)
```

Comment:
| Before 🐛 | After 🦋 |
| --- | --- |
| <img width="460" alt="Screen Shot 2019-06-25 at 5 50 46 PM" src="https://user-images.githubusercontent.com/387596/60143828-ed046700-9774-11e9-8a47-a129c87a7c9a.png"> | <img width="460" alt="Screen Shot 2019-06-25 at 5 49 33 PM" src="https://user-images.githubusercontent.com/387596/60143840-fdb4dd00-9774-11e9-97b8-97682d8a053f.png"> |

This constraint caused the button to align to the edge of the stack view. This was easily reproducible if you made dynamic type font large and open any pledge detail screen.

Reply:
I did not reproduce this but it could have been pointed out in https://github.com/kickstarter/ios-oss/pull/702.